app/views/katello/subscription_mailer/subscription_expiry.text.erb

```diff
@@ -13,3 +13,5 @@
   <%= _("Days Remaining") %>:       <%= pool.days_until_expiration %>
 
 <% end %>
+
+<%= _("Report") %>:               <%= report_url %>
```

Comment:
To me this is almost too pithy.  What about 
* "Full report"
* "View the full report"
* "View the full report, including affected hosts"

Thoughts?

Reply:
+1 to 'view the full report'

Reply:
yes. "View the full report" sounds good. thanks!